java/src/org/openqa/selenium/remote/service/DriverFinder.java

```diff
@@ -96,17 +96,20 @@ private Result getBinaryPaths() {
         String driverName = service.getDriverName();
         result = new Result(service.getExecutable());
         if (result.getDriverPath() == null) {
-          result = new Result(System.getProperty(service.getDriverProperty()));
+          result = new Result(System.getenv(service.getDriverEnvironmentVariable()));
```

Comment:
Trying to think if we care about null pointer exception here. DriverService returns `null`, but I can't think of a scenario where you would use DriverFinder with anything that isn't one of these defined services.

Reply:
actually, never mind, this is the same as what we have for getProperty already...